### PR TITLE
idlharness test fixups

### PR DESCRIPTION
**Pull request by stephenmcgruer**
These are follow-ups from https://github.com/web-platform-tests/wpt/pull/23397#issuecomment-624203679

### CODE REVIEW

**Comment by Ms2ger** [comment]
Why wasn't this included in #23397?

Is https://github.com/web-platform-tests/wpt/issues/12850 fixed now?

**Comment by stephenmcgruer** [comment]
> Why wasn't this included in #23397?

To avoid conflating rolling a third_party library with broken tests. In retrospect, I should have included both commits in the same PR - I forgot that GitHub has a rebase-and-merge model as well as a squash one -_-. (Too much time working in Chromium).

> Is #12850 fixed now?

It was already fixed, by c46ec78846335260f32954bcafa1676d600836f0

**Comment by stephenmcgruer** [comment]
cc @foolip - note that trusted-types appears to have hand-written idl in `interfaces/` - can it (and should it) be added to reffy?

File: interfaces/trusted-types.tentative.idl [hunk 1]
```diff
@@ -29,7 +29,7 @@ typedef [StringContext=TrustedScriptURL] USVString ScriptURLString;
   Exposed=(Window, Worker),
   SecureContext
 ] interface TrustedTypePolicyFactory {
-    TrustedTypePolicy createPolicy(DOMString policyName, TrustedTypePolicyOptions policyOptions);
+    TrustedTypePolicy createPolicy(DOMString policyName, optional TrustedTypePolicyOptions policyOptions);
```
Comment (domenic) [inline interfaces/trusted-types.tentative.idl:32]:
```suggestion
    TrustedTypePolicy createPolicy(DOMString policyName, optional TrustedTypePolicyOptions policyOptions = {});
```

(without the `= {}` is invalid Web IDL these days; I imagine a future webidl2.js roll might start erroring there.)

Comment (stephenmcgruer) [reply interfaces/trusted-types.tentative.idl]: Done, thanks.

**Review by domenic — APPROVED** [review]

**Pull request merged**